index.html

```diff
@@ -170,6 +170,33 @@ <h2 class="letsconnect2">LET'S CONNECT</h2>
     </li>
     </ul>
     </section>
-  
+
+    <div class="foot-back">
+    <footer> 
```

Comment:
- [ ] kindly fix the overflow in your website to make it looks awesome and prevent users from scrolling right and left

![formerrrrrrrr](https://user-images.githubusercontent.com/89282221/187549545-fe26164c-e3c6-4162-a0f4-b32b6c68aecf.gif)